mayastor/src/bdev/nexus/nexus_child.rs

```diff
@@ -439,4 +439,16 @@ impl NexusChild {
             .map(|j| j.stats().progress as i32)
             .unwrap_or_else(|| -1)
     }
+
+    /// Returns true if the child is on the same node as the nexus.
```

Comment:
Returns Some(true) or None if the bdev is not open?

Reply:
I have updated the comments - thanks